samples/account-linking/csharp/.tours/tab-account-linking.tour

```diff
@@ -0,0 +1,221 @@
+    {
+      "file": "Source/wwwroot/scripts.js",
+      "description": "We then call the `/github/repositories` API on our web API with the `Bearer` token attached as the authorization header per the [OAuth2.0 specification](https://datatracker.ietf.org/doc/html/rfc6750#section-2.1). ",
+      "line": 58,
+      "selection": {
+        "start": {
+          "line": 53,
+          "character": 1
+        },
+        "end": {
+          "line": 58,
+          "character": 6
+        }
+      }
+    },
+    {
+      "file": "Source/Controllers/GitHubController.cs",
+      "description": "# Getting a verifiable identity\r\nThe call then lands on the backend where we use the [Microsoft.Identity.Web](https://docs.microsoft.com/en-us/azure/active-directory/develop/microsoft-identity-web) library to validate the bearer token & parse out claims. We use the claims in the token as they are signed by AzureAD and are therefore more trustworthy than a url parameter or similar unauthenticated parameter.\r\n\r\nWe then grab the user and tenant ids from the token which we will use as the key in our database of GitHub access and refresh tokens.\r\n",
+      "line": 36,
+      "selection": {
+        "start": {
+          "line": 35,
+          "character": 1
+        },
+        "end": {
+          "line": 36,
+          "character": 69
+        }
+      }
+    },
+    {
+      "file": "Source/Controllers/GitHubController.cs",
+      "description": "## Attempt 1 at getting the User's GitHub access token\r\nWe then use the `OAuthTokenProvider` service to attempt to get the access token for the current user. This will return a `NeedsConsentResult` as the database isn't populated with the current user's GitHub access & refresh tokens. ",
+      "line": 46,
+      "selection": {
+        "start": {
+          "line": 46,
+          "character": 9
+        },
+        "end": {
+          "line": 46,
+          "character": 104
+        }
+      }
+    },
+    {
+      "file": "Source/Controllers/GitHubController.cs",
+      "description": "# Redirecting the Tab to the auth flow\r\nWhen the `OAuthTokenProvider` doesn't have a token for the user, it generates a cryptographically tamper evident `state` object and crafts the redirection URI to this app's auth start.\r\n\r\nWe will go into detail of how the `state` is generated and used in the [token storage tour][Token Storage].",
+      "line": 56,
+      "selection": {
+        "start": {
+          "line": 48,
+          "character": 8
+        },
+        "end": {
+          "line": 56,
+          "character": 15
+        }
+      }
+    },
+    {
+      "file": "Source/Controllers/GitHubController.cs",
+      "description": "We return back an easily identified status code, in this case `HTTP 412/PreconditionFailed` so that the tab can know to raise the authorization prompt.\r\n\r\nWe attach the `state` object so we can correlate the user's login/consent for in GitHub with the user who attempted this request.  ",
```

Comment:
if we don't have a header here (might need to be first) codetour just uses the file path as the name in the sidebar. And perhaps worse, the Next button is a smaller target, and that hurts my brain :D.
(super nitty, but might be good to try and summarize the step)